### ISSUE TITLE
qwen 2.7B MOE 训练缓慢，GPU 利用率很低。另外，断点续训时，预估的剩余时间需要休修正下

### ISSUE THREAD

**Issue by XiaoMaGe-hero** [modelscope/ms-swift#1732]
**Describe the bug**

{'loss': 0.17637376, 'acc': 0.93708344, 'grad_norm': 2.34915471, 'learning_rate': 9.043e-05, 'memory(GiB)': 46.92, 'train_speed(iter/s)': 9.176681, 'epoch': 0.24, 'global_step/max_steps': '2005/8325', 'percentage': '24.08%', 'elapsed_time': '3m 34s', 'remaining_time': '11m 16s'}
{'loss': 0.15013269, 'acc': 0.94435329, 'grad_norm': 1.5036732, 'learning_rate': 9.038e-05, 'memory(GiB)': 49.99, 'train_speed(iter/s)': 4.655427, 'epoch': 0.24, 'global_step/max_steps': '2010/8325', 'percentage': '24.14%', 'elapsed_time': '7m 7s', 'remaining_time': '22m 23s'}
{'loss': 0.24201331, 'acc': 0.93665237, 'grad_norm': 1.41412926, 'learning_rate': 9.032e-05, 'memory(GiB)': 49.99, 'train_speed(iter/s)': 3.106456, 'epoch': 0.24, 'global_step/max_steps': '2015/8325', 'percentage': '24.20%', 'elapsed_time': '10m 44s', 'remaining_time': '33m 38s'}
{'loss': 0.20011899, 'acc': 0.93291664, 'grad_norm': nan, 'learning_rate': 9.027e-05, 'memory(GiB)': 49.99, 'train_speed(iter/s)': 2.341594, 'epoch': 0.24, 'global_step/max_steps': '2020/8325', 'percentage': '24.26%', 'elapsed_time': '14m 18s', 'remaining_time': '44m 40s'}
{'loss': 0.16978382, 'acc': 0.94291668, 'grad_norm': 4.87751007, 'learning_rate': 9.021e-05, 'memory(GiB)': 49.99, 'train_speed(iter/s)': 1.879459, 'epoch': 0.24, 'global_step/max_steps': '2025/8325', 'percentage': '24.32%', 'elapsed_time': '17m 53s', 'remaining_time': '55m 39s'}
{'loss': 0.25152452, 'acc': 0.93478451, 'grad_norm': 3.49017692, 'learning_rate': 9.015e-05, 'memory(GiB)': 49.99, 'train_speed(iter/s)': 1.571546, 'epoch': 0.24, 'global_step/max_steps': '2030/8325', 'percentage': '24.38%', 'elapsed_time': '21m 27s', 'remaining_time': '1h 6m 33s'}
{'loss': 0.18838446, 'acc': 0.94577446, 'grad_norm': 2.32465887, 'learning_rate': 9.009e-05, 'memory(GiB)': 51.54, 'train_speed(iter/s)': 1.352303, 'epoch': 0.24, 'global_step/max_steps': '2035/8325', 'percentage': '24.44%', 'elapsed_time': '25m 0s', 'remaining_time': '1h 17m 19s'}
{'loss': 0.17079583, 'acc': 0.9489584, 'grad_norm': 2.7604537, 'learning_rate': 9.003e-05, 'memory(GiB)': 51.54, 'train_speed(iter/s)': 1.187178, 'epoch': 0.25, 'global_step/max_steps': '2040/8325', 'percentage': '24.50%', 'elapsed_time': '28m 34s', 'remaining_time': '1h 28m 1s'}
{'loss': 0.2085937, 'acc': 0.93831854, 'grad_norm': 3.20240998, 'learning_rate': 8.997e-05, 'memory(GiB)': 51.54, 'train_speed(iter/s)': 1.057665, 'epoch': 0.25, 'global_step/max_steps': '2045/8325', 'percentage': '24.56%', 'elapsed_time': '32m 9s', 'remaining_time': '1h 38m 45s'}
{'loss': 0.18175808, 'acc': 0.93770819, 'grad_norm': 2.57533073, 'learning_rate': 8.991e-05, 'memory(GiB)': 51.54, 'train_speed(iter/s)': 0.949554, 'epoch': 0.25, 'global_step/max_steps': '2050/8325', 'percentage': '24.62%', 'elapsed_time': '35m 54s', 'remaining_time': '1h 49m 56s'}
{'loss': 0.22696259, 'acc': 0.93395844, 'grad_norm': 3.58553958, 'learning_rate': 8.985e-05, 'memory(GiB)': 51.54, 'train_speed(iter/s)': 0.866321, 'epoch': 0.25, 'global_step/max_steps': '2055/8325', 'percentage': '24.68%', 'elapsed_time': '39m 28s', 'remaining_time': '2h 0m 25s'}
{'loss': 0.16824765, 'acc': 0.9645833, 'grad_norm': 3.27143717, 'learning_rate': 8.979e-05, 'memory(GiB)': 51.54, 'train_speed(iter/s)': 0.796255, 'epoch': 0.25, 'global_step/max_steps': '2060/8325', 'percentage': '24.74%', 'elapsed_time': '43m 3s', 'remaining_time': '2h 10m 55s'}
{'loss': 0.19189042, 'acc': 0.9458334, 'grad_norm': 3.96856308, 'learning_rate': 8.973e-05, 'memory(GiB)': 51.54, 'train_speed(iter/s)': 0.732101, 'epoch': 0.25, 'global_step/max_steps': '2065/8325', 'percentage': '24.80%', 'elapsed_time': '46m 56s', 'remaining_time': '2h 22m 18s'}
{'loss': 0.17862873, 'acc': 0.93812504, 'grad_norm': 3.50377727, 'learning_rate': 8.967e-05, 'memory(GiB)': 51.54, 'train_speed(iter/s)': 0.679539, 'epoch': 0.25, 'global_step/max_steps': '2070/8325', 'percentage': '24.86%', 'elapsed_time': '50m 42s', 'remaining_time': '2h 33m 12s'}
{'loss': 0.19685781, 'acc': 0.95687504, 'grad_norm': 2.08957148, 'learning_rate': 8.961e-05, 'memory(GiB)': 51.54, 'train_speed(iter/s)': 0.636832, 'epoch': 0.25, 'global_step/max_steps': '2075/8325', 'percentage': '24.92%', 'elapsed_time': '54m 14s', 'remaining_time': '2h 43m 22s'}
{'loss': 0.15695015, 'acc': 0.94971819, 'grad_norm': 2.60611653, 'learning_rate': 8.955e-05, 'memory(GiB)': 51.54, 'train_speed(iter/s)': 0.599372, 'epoch': 0.25, 'global_step/max_steps': '2080/8325', 'percentage': '24.98%', 'elapsed_time': '57m 46s', 'remaining_time': '2h 53m 27s'}
Train:  25%|██████████████████████████████████▉                                                                                                         | 2081/8325 [58:28<73:44:49, 42.52s/it] 

**Your hardware and system info**

24vCPU 192G 2*Tesla-V100-32G , GPU：Tesla-V100-32G

**Additional context**

gpu 内存利用率在 60%-80%， 但是两台gpu利用率波动模型，且利用率很低。

**Comment by tastelikefeet** [comment]
贴一下运行命令呗
